version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.4.9-SNAPSHOT"
+version in ThisBuild := "0.4.10-SNAPSHOT"
```

Comment:
Not sure myself: What's the convention here?  Is it incremented when the result schema changes?